src/pages/CreateAccount.js

```diff
@@ -0,0 +1,230 @@
+import { ACCOUNT_ID_SUFFIX, handleCreateAccount } from '../utils/auth';
+import { getEmailId, isValidEmail, parseURLParams } from '../utils/generic';
+
+import React from 'react';
+import styled from 'styled-components';
+import { useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+
+const CreateAccount = () => {
+  const history = useHistory();
+  const [urlParams, setUrlParams] = React.useState(null);
+  const [isAccountAvailable, setIsAccountAvailable] = React.useState(null);
+  const { register, handleSubmit, watch, setValue, formState } = useForm();
+  const formValues = watch();
+  const defaultAccountId = getEmailId(formValues?.email || '')
+
+  const checkIsAccountAvailable = async () => {
+    try {
+      if (!formValues?.accountId) return
+
+      const response = await fetch('https://rpc.testnet.near.org', {
```

Comment:
Not a blocker for testnet testing, but this should be referencing the RPC url from our NEAR configuration rather than hardcoded